src/js/edu-benefits/utils/validations.js

```diff
@@ -268,7 +268,7 @@ function isValidTourOfDuty(tour) {
 }
 
 function isValidMilitaryServicePage(data) {
-  return !data.chapter33 || isNotBlank(data.benefitsRelinquished.value);
+  return isBlank(data.serviceAcademyGraduationYear.value) || isValidYear(data.serviceAcademyGraduationYear.value);
```

Comment:
The MilitaryServicePage is using `isValidCurrentorPastYear` for the validation on the `serviceAcademyGraduationYear` itself, so we should use the same validation here too so it's consistent.